src/App.tsx

```diff
@@ -8,7 +7,93 @@ import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
+import { Todo } from './types/Todo';
+import { SortType } from './types/SortType';
+import { getTodos } from './api';
+
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [visibleTodos, setVisibleTodos] = useState<Todo[]>([]);
+  const [filteredBySelect, setFilteredBySelect] = useState(SortType.all);
+  const [filteredByQuery, setFilteredByQuery] = useState('');
+  const [selectedTodo, setSelectedTodo] = useState<Todo | null>(null);
+  const [showTodoModal, setShowTodoModal] = useState<boolean>(false);
```

Comment:
```suggestion
  const [showTodoModal, setShowTodoModal] = useState(false);
```
not need to pass it for boolean (same as numbers and strings)